setup.py

```diff
@@ -86,7 +86,6 @@ def get_extras_require() -> Dict[str, List[str]]:
         "testing": [
             # TODO(toshihikoyanase): Remove the version constraint after resolving the issue
             # https://github.com/optuna/optuna/issues/1000.
```

Comment:
Can we remove these lines since it issue refers to `bokeh` that will be removed in this PR?

Reply:
Thank you. I removed in 80df813.

Reply:
still in https://github.com/optuna/optuna/blob/80df8136168e283a7dab7d93ac6f5ad51bc2ded5/setup.py#L87-L88

Reply:
:bow: 59d11ea